File: web-app/src/app/core/models/sync-log-entry.ts
export interface SyncLogEntry {
  id: string;
  userId: number;
  accountId: number;
  exchangeName: string;
  orderId: string;
  symbol: string;
  side: string;
  qty: number;
  price: number;
  status: string;
  errorMessage: string | null;
  timestamp: string;
  importSource: string;
}

File: api/api/Exchanges/Models/SyncLogEntry.cs
namespace api.Exchanges.Models;

public record SyncLogEntry(
    string Id,
    int UserId,
    int AccountId,
    string ExchangeName,
    string OrderId,
    string Symbol,
    string Side,
    decimal Qty,
    decimal Price,
    string Status,
    string? ErrorMessage,
    DateTime Timestamp,
    string ImportSource);
